website_membership_gamification/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2022 Manuel Regidor <manuel.regidor@sygel.es>
+# Copyright 2023 Tecnativa - Pedro M. Baeza
+# Copyright 2023 Tecnativa - Carolina Fernandez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Website Membership Gamification",
+    "summary": "Show badges assigned to users on website",
+    "version": "18.0.1.0.0",
+    "category": "Website",
+    "author": "Sygel, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/vertical-association",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "development_status": "Beta",
+    "depends": ["hr_gamification", "website_membership"],
```

Comment:
```suggestion
    "depends": ["gamification", "website_membership"],
```
The dependency must remain `gamification`.